### ISSUE TITLE
[Feature request] Official magisk discord server

### ISSUE THREAD

**Issue by Anty0** [topjohnwu/Magisk#2090]
## Suggestion:
Create official magisk discord server.

## Expectations:
Community can help other newcomers to fix common issues with magisk - less spam for topjohnwu (useless issues, etc.).

## Why?
Using XDA forum as only official social thing for such big project does not feel enough. I can see lot of people loosing interest in this community, just because there isn't enough space for open communication.

### Tl;Dr
Let's create official magisk discord server!

**Comment by Didgeridoohan** [comment]
One more thing to maintain and more channels of communication to keep track of?

I vote nay...

**Comment by palmbeach05** [comment]
Xda github twitter, i think we got enough.

**Comment by diareuse** [comment]
I'd personally ignore Discord as well as I ignore XDA, Twitter and DMs.
This would only add so much additional garbage. 

I suggest you creating unofficial one, if you so need it, and elect someone who **checks duplicates/fakes/general garbage** and reports filtered issues on Github. _You might as well do the same for Telegram._ Only then it would be useful channel. Forcing already busy mods on new platform is imm not a good approach. 🙂

**Comment by GTHSS** [comment]
i want a magisk discord server because that would make magisk cooler

**Comment by SX-9** [comment]
looks like we have [this server](https://discord.gg/androidroot) from the r/AndroidRoot subreddit
